CIScripts/Tests/VRouterAgentTests.ps1

```diff
@@ -398,12 +505,54 @@ function Test-VRouterAgentIntegration {
         Write-Host "===> PASSED: Test-Pkt0ReceivesTrafficAfterAgentIsStarted"
     }
 
-    Test-InitialPkt0Injection -Session $Session -TestConfiguration $TestConfiguration
-    Test-Pkt0RemainsInjectedAfterAgentStops -Session $Session -TestConfiguration $TestConfiguration
-    Test-OnePkt0ExistsAfterAgentIsRestarted -Session $Session -TestConfiguration $TestConfiguration
-    Test-Pkt0ReceivesTrafficAfterAgentIsStarted -Session $Session -TestConfiguration $TestConfiguration
-    Test-GatewayArpIsResolvedInAgent -Session $Session -TestConfiguration $TestConfiguration
+    function Test-SingleComputeNodePing {
+        Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session,
+               [Parameter(Mandatory = $true)] [TestConfiguration] $TestConfiguration)
+        Write-Host "===> Running: Test-SingleComputeNodePing"
+        Test-Ping -Session1 $Session -Session2 $Session -TestConfiguration $TestConfiguration -Container1Name "jolly_lumberjack" -Container2Name "juniper_tree"
```

Comment:
Could we use more generic container names? For example `container1` and `container2`.